timemachine/cpp/src/kernels/k_nonbonded_pair_list.cuh

```diff
@@ -193,4 +182,10 @@ void __global__ k_nonbonded_pair_list(
             accumulate<Negated>(du_dp + w_param_idx_j, g_wj);
         }
     }
+    // Always accumulate into the energy buffers even if there is no interaction to ensure
+    // buffer is zeroed out, avoids having to memset every call
+    if (u_buffer) {
+        // Do not do `FLOAT_TO_FIXED_ENERGY(Negated ? -u : u)` as that can produce a positive fixed point value
+        u_buffer[pair_idx] = Negated ? -FLOAT_TO_FIXED_ENERGY<RealType>(u) : FLOAT_TO_FIXED_ENERGY<RealType>(u);
+    }
```

Comment:
track for future PR: consider renaming `NonbondedPairListNegated` in the Python API to `NonbondedExclusions` or similar, to emphasize that it is specialized for exclusions and not valid on its own